docker-compose.yml

```diff
@@ -47,8 +50,5 @@ services:
       - "27017:27017"
   clearlydefined_mongo_seed:
     image: "clearlydefined/docker_dev_env_experiment_clearlydefined_mongo_seed"
-    build:
-      context: ./mongo_seed
-      dockerfile: Dockerfile
```

Comment:
Why this change? 🤔

Reply:
Building this clearlydefined_mongo_seed image through docker-compose on a windows system throws error as "**ERROR [clearlydefined_mongo_seed internal] load build context**". The usage of building this image was to add and test specific seed data and it is not necessary for the regular dev environment setup. Hence removed this section from the docker-compose.yml file so that there are no issues in setting up dev environment in any of the OS.

Reply:
Hi @yashkohli88.  At our dev meeting, I mentioned that it seems fine to remove the build if you add instructions in the README about how to run it manually.  Let me know when that is done and we'll get this merged.

Reply:
@elrayle @yashkohli88 There is an existing [PR](https://github.com/clearlydefined/docker_dev_env_experiment/pull/12/files) documenting how to build and publish mongo_seed image.  Maybe we can add onto that PR to incorporate suggestions?